scripts/dev_tools.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     command="xdg-open";;
+    Darwin*)    command="open";;
+    *)          machine="UNKNOWN:${unameOut}"
+esac
+echo ${command}
 
 if [[ -z "$1" ]]; then
     # list db, redis, rabbitmq, and seaweedfs ports
     docker ps --format '{{.Names}}' | grep "db\|redis\|queue\|seaweedfs" | xargs -I '{}' bash -c 'echo "{} -> $(python3 -c "import sys; [print(l.split(\":\")[-1]) for l in sys.argv[1].split(\"\n\") if l.split(\"/tcp\")[0] in sys.argv[2].split(\",\")]" "$(docker port {})" "15672,5432,6379,8888")"' - {}
 else
     PORT=$1
     if docker ps | grep ":${PORT}" | grep -q 'redis'; then
-        open redis://127.0.0.1:${PORT}
+        ${command} redis://127.0.0.1:${PORT}
     elif docker ps | grep ":${PORT}" | grep -q 'postgres'; then
-        open postgresql://postgres:changethis@127.0.0.1:${PORT}/app
+        ${command} postgresql://postgres:changethis@127.0.0.1:${PORT}/app
     else
-        open http://localhost:${PORT}
+        ${command} http://localhost:${PORT}
```

Comment:
Does TablePlus also open in Linux with xdg-open command?

Reply:
It does. But you have to associate the url with TablePlus using xdg-mime.

Reply:
Example: `xdg-mime default tableplus.desktop x-scheme-handler/postgresql`

Reply:
I see Wonderful

Reply:
@rAlexandre00 awesome PR thank you so much. Can we add this scheme association command in a comment at the top of the script for others to use?